### PR TITLE
[0.25deg] Update PE layout

### PR DESCRIPTION
**Pull request by minghangli-uni**
See https://github.com/COSIMA/access-om3/issues/214

### CODE REVIEW

File: nuopc.runconfig [hunk 1]
```diff
@@ -40,31 +40,31 @@ PELAYOUT_attributes::
      esp_nthreads = 1
      esp_pestride = 1
      esp_rootpe = 0
-     glc_ntasks = 240
+     glc_ntasks = 1
      glc_nthreads = 1
      glc_pestride = 1
      glc_rootpe = 0
-     ice_ntasks = 240
+     ice_ntasks = 96 #Note-Parallel I/O github.com/COSIMA/access-om3/issues/214
```
Comment (anton-seaice) [inline nuopc.runconfig:47]: I think if https://github.com/payu-org/payu/pull/496 gets merged we don't need this note (which is technically also true for all the model components.)

File: nuopc.runconfig [hunk 2]
```diff
@@ -27,11 +27,11 @@ DRIVER_attributes::
 ::
 
 PELAYOUT_attributes::
-     atm_ntasks = 240
+     atm_ntasks = 48
      atm_nthreads = 1
      atm_pestride = 1
      atm_rootpe = 0
-     cpl_ntasks = 240
+     cpl_ntasks = 96
```
[the remark this answers is not in the document]
Comment (minghangli-uni) [reply nuopc.runconfig:34]: Yes this is intentional. 96 cores for cpl and 48 cores are for data components.

**Comment by anton-seaice** [comment]
I ran this for one month and it works :)

**Review by dougiesquire — APPROVED** [review]
I don't know enough about the cice block stuff to be useful, but the PELAYOUT looks good to me. Thanks both!

**Comment by dougiesquire** [comment]
@minghangli-uni, I presume this supersedes https://github.com/ACCESS-NRI/access-om3-configs/pull/60 so we can close that?

**Comment by minghangli-uni** [comment]
Right! Thanks @dougiesquire

**Pull request merged**